src/renderer/Settings.tsx

```diff
@@ -145,11 +145,13 @@ function URLInput({ initialURL, onValidURL }: URLInputProps) {
 	}, [initialURL]);
 
 	function onChange(event: React.ChangeEvent<HTMLInputElement>) {
-		setCurrentURL(event.target.value);
+		let eventValue = event.target.value.trim();
```

Comment:
since `eventValue` won't change, I think it should be a `const` instead of `let`.